cmd/agent/app/options/options_test.go

```diff
@@ -0,0 +1,151 @@
+package options
```

Comment:
Missing license header here.

I think `cmd/server/app/options/options_test.go` might be missing license header too.

Reply:
Sadly more than just these two. Did a quick scan and added license headers where I found them to be missing.